src/t8_schemes/t8_default/t8_default_quad/t8_default_quad_cxx.cxx

```diff
@@ -799,7 +799,18 @@ t8_default_scheme_quad_c::t8_element_reference_coords (const t8_element_t
                                                        double *out_coords)
   const
 {
-  SC_ABORTF ("Not implemented\n");
+  T8_ASSERT (t8_element_is_valid (elem));
```

Comment:
Please move this implementation to a C function `t8_dquad_compute_reference_coords`. For this you will have to introduce new files `t8_dquad_bits.{h,c}`, which is fine since we want to have them in the near future anyway.

Reply:
Done